packages/model/src/working-group.ts

```diff
@@ -33,7 +33,7 @@ export type WorkingGroupDataObject = {
   externalLinkText?: string;
   members: WorkingGroupMember[];
   pointOfContact?: WorkingGroupMember;
-  complete: boolean;
+  complete?: boolean;
```

Comment:
can't we remove this `?` in here? 

since it's being used `!!` in `complete: !!workingGroupGraphQl.flatData.complete` I think there will always be a boolean value, right?